content/posts/macinplanck-configuration/index.md

```diff
@@ -0,0 +1,326 @@
+
+```markdown
+| Layer        | Color                             |
+| ------------ | --------------------------------- |
+| `Lower 0`    | `#ecb392` (Raspberry Light Brown) |
+| `Base 1`     | `#f2671f` (Sunset Orange)         |
+| `Raise 2`    | `#219e20` (Material Green)        |
+| `Adjust 3`   | `#d91f89` (Sunset Pink)           |
+| `NORMAL 4`   | `#00a3e9` (Desert Sun Blue)       |
+| `VISUAL 5`   | `#878eff` (Solarized Purple)      |
+| `NRMLSHFT 6` | `#219e20` (Material Green)        |
+| `WINDOWZ 7`  | `#2e00e9` (Sunset Blue/Purple)    |
+| `LGUI_ALL 8` | `#ea1e63` (Material Red)          |
+```
+
+`NRMLSHIFT` and `Raise` having the same color as the dominant color is an accident that I'm too lazy to fix. They'll never be accessible from the same layer, so I think that it's fine.
+
+## Feedback (if anyone is listening)
+
+After going through all of this, I've got some feedback.
+
+- The Wally GUI application was a tad more unreliable than I would've liked. Sometimes I would go to flash the board, the app would hang, and then _every keyboard I used wouldn't work until I rebooted_. The `wally-cli` CLI tool was rock solid, though, and I've been using that for all of my flashing. Your mileage may vary.
+- Especially when I was in the midst of changing colors, I found it basically impossible to tell what color was already used from the per-key palettes when you selected a key. It'd be great if, when you selected a key in the UI, it'd also tell you what color you've assigned to that key.
+- The `OSL()` layer action can bring you to a layer, but it disables the key in that layer. This makes sense for layer actions like `MO()`, because the layer is only active when you are actively holding the key that got you there. However, for layer functions like `OSL()`, it's perfectly fine to lift your finger off the key that got you into that layer, freeing that key up for other actions. This made things like a `gg` motion in the `NORMAL` layer (which I would've mapped to `cmd+up`) much more clumsy to implement with the UI, so I didn't bother.
+
+## What's next?
+
+There are already multiple things that I want to improve upon for this layout.
+
+- Add some more actions into the `Adjust` layer; I basically ignored it heavily because I couldn't think of anything, so this is ripe for improvement.
+- Come up with some more common Vim motions that I can accomplish without getting into macros; at a certain point, getting into macros will be inevitable, but using the GUI tool is so much easier than editing the keymap code (IMO) that I'm going to dread when I have to switch over.
+- Once I exhaust all available options to me in the UI, I want to start messing around with creating more complex [macros](https://docs.qmk.fm/#/feature_macros). There were a few chords/motions that I created that definitely could have been more cleanly defined via macros, and that's probably the next thing that I'd get around to messing aroun
```

Comment:
> . . . thing that I'd get around to messing aroun

Clipped sentence and duplicated word it looks like.

Reply:
Cleaned that up, oops.